### ISSUE TITLE
monsuta

### ISSUE THREAD

**Issue by lipamanka** [lipu-linku/pali-nimi#24]
#### sona Linku pi toki Inli

fear, dread; monster, predator; threat, danger

#### sona Linku pi toki pona

pilin ike pi ken moli; pilin ike pi ken pakala; ijo ni: ona li pana e pilin ni

#### sona ku

scary³, fear³, monster³, horror³, scare², disturbing², terror², beast², afraid², demon²

#### sona sin

**Comment by janPensa** [comment]
Here are my main thoughts on the "monsuta" definition.


I think the definition for "monsuta" should not include "fear, dread", because in my experience, it does not reflect actual usage. People who frequently use the word "monsuta" generally do not use it on its own to mean "fear". They usually use "pilin monsuta" for "fear", and they use "monsuta" for "scary, creepy, causing fear" or "monster, something that causes fear".

A lot of Toki Pona dictionaries include "fear" as one of the primary meanings of "monsuta", which causes a lot of learners to use the word in ways that aren't in line with how more experienced users of the word tend to use it.


In my opinion, both "monster" and "scary" should definitely be in the Linku definition. I'm not sure which of the two I would put first. Other translations worth considering to include:

> something that causes fear, frightening, terrifying, creepy, spooky, eerie

Translations _maybe_ worth considering:

> scary creature, monstrosity, monstrous, monster-like, demon, predator, danger, threat, dangerous, threatening


Personally, I would _not_ include translations of transitive uses of "monsuta" (such as "to scare", "to fear"), because transitive monsuta is poorly defined in current usage and those translations can be confusing to learners. Most of the ku translations can also be misleading for similar reasons (e.g. "beast" can be used as a synonym of monster, but can also be taken to mean something closer to "animal"), or aren't reflective of community consensus (e.g. using "monsuta" for "afraid")

**Comment by lipamanka** [comment]
> Personally, I would not include translations of transitive uses of "monsuta" (such as "to scare", "to fear"), because transitive monsuta is poorly defined in current usage and those translations can be confusing to learners.

Yeah to add on here a large part of pedagogy happens outside of small linku definitions so if we just don't put anything here it gives a teacher more room to explain usage, which is important for monsuta because it's particularly fucky transitively

**Comment by jnpoJuwan** [comment]
the transitive uses of 'monsuta' could be explained in the usage notes/commentary. it feels wrong to me to leave out an important usage of it for teachers to explain, when students or the teachers themselves (not everyone knows everything) could learn about it independentely

basing off of a post by kala pona Tonyu on lipu kule (https://lipukule.org/post/2021/01/29/monsutatesu/):
> 'monsuta' as a transitive verb has two opposite meanings: 'to frighten' and 'to turn into a monster' or, in short, 'to fear'.

in her post, kala pona Tonyu also states that 'kalama', 'mama', 'moku' and 'lawa' are also funky transitively, so that may be something to keep in mind for their new definitions
